as-http.js

```diff
@@ -61,7 +61,8 @@ TCurlAsHttp.prototype.send = function send() {
         streamed: true,
         hasNoParent: true
     });
-    self.asHttpClient.sendRequest(req, hreq, onSent);
+    self.asHttpClient.channel = self.subChannel.topChannel;
```

Comment:
btw, it's _supposed_ to get it from the req on first use, but we missed that in the "normal" send path; should be done like the forwarding path: https://github.com/uber/tchannel-node/blob/master/as/http.js#L341